.pylintrc

```diff
@@ -1,3 +1,6 @@
 [MESSAGES CONTROL]
 disable=line-too-long,
-	missing-docstring
\ No newline at end of file
+	missing-docstring
+
+[TYPECHECK]
+ignored-modules=distutils,distutils.core
```

Comment:
pylon running on travis inside virtualenv has issues with the setup.py file not being able to import distils.core https://github.com/PyCQA/pylint/issues/73